src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -525,6 +526,9 @@ export class GlobalMap extends React.Component<Props, State> {
   }
 
   onRegionIsChanging = async () => {
+    if (!this.map) {
```

Comment:
Curious about this change – seems like there's a story behind it?

Reply:
Ahh yes. I had meant to add a comment here about it after submitting the pull request, but forgot about it after standup 😬 

I was seeing `unhandled promise rejection`s pointing to this line when panning around in the map. It seemed that the map ref was `undefined` at some points during panning, but not all the time. As far as I could tell the map was behaving correctly despite these errors, so I wasn't sure it was worth digging into right now, but we should probably add a ticket to follow up on it since it could be a symptom of a larger issue. What do you think?

Reply:
Hmm, that's a good question. I think this change makes sense, even without the promise rejections you were seeing, but I'm not sure if it's worth digging in further.

@l2succes what do you think?

Reply:
Hmm I think this might need further investigation after all. Seeing things like this now: 
![image](https://user-images.githubusercontent.com/1242537/60674714-a4cefe00-9e72-11e9-97f3-401731696094.png) which is related to the same 'map ref is undefined sometimes' error, and should probably not be happening.

I checked and it's happening on master for me, so it's unrelated to the changes in this PR. One thing I noticed that might be relevant is this:

![image](https://user-images.githubusercontent.com/1242537/60679963-786eae00-9e81-11e9-8d8c-28b6dddbc02f.png)